__tests__/api.test.js

```diff
@@ -1,22 +1,8 @@
 const app = require('../app.js');
 const request = require('supertest');
-const db = require("../db/connection");
-const seed = require("../db/seeds/seed");
-const data = require("../db/data/test-data/index");
-const format = require("pg-format");
```

Comment:
In your previous comments, as far as I can see the comment meant to only remove the unnecessary elements (i.e. pg-format). The seeding,data and connection and afterAll/beforeEach should still be needed